vispy/scene/widgets/widget.py

```diff
@@ -63,9 +63,13 @@ def __init__(self, pos=(0, 0), size=(10, 10), border_color=None,
         self._size = 100, 100
 
         # layout interaction
-        # todo: use Cassowary; see #277
-        self._fixed_size = (None, None)
+        self._width_limits = (1, None)
```

Comment:
keeping this as a tuple introduces a decent number of annoyances, but I chose the annoyance over having two separate fields (`min` and `max`). Should I change this? I'm unable to decide